pkg/handlers/generic/mutation/users/tests/generate_patches.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/utils/ptr"
+	runtimehooksv1 "sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
+
+	"github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/api/v1alpha1"
+	"github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/common/pkg/capi/clustertopology/handlers/mutation"
+	"github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/common/pkg/testutils/capitest"
+	"github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/common/pkg/testutils/capitest/request"
+)
+
+var (
+	testUser1 = v1alpha1.User{
+		Name:   "complete",
+		Passwd: ptr.To[string]("password"),
+		SSHAuthorizedKeys: []string{
+			"key1",
+			"key2",
+		},
+		Sudo: ptr.To[string]("ALL=(ALL) NOPASSWD:ALL"),
```

Comment:
Type should be inferrable
```suggestion
		Sudo: ptr.To("ALL=(ALL) NOPASSWD:ALL"),
```